css/style.css

```diff
@@ -80,3 +79,61 @@ body {
 .name {
   font-weight: 500;
   font-size: 1.2rem; }
+
+  .container {
+    width: 100%;
+    height: 100vh;
+    display: grid;
+    place-items: center;
+  }
+.toggle {
+    width: 6rem;
+    height: 6rem;
+    background-color: #eee8e1;
+    position: absolute;
+    top: 2rem;
+    right: 2rem;
+    display: grid;
+    place-items: center;
+    border-radius: 50%;
+    cursor: pointer;
+  }
+  .container.dark .stopwatch .toggle {
+    background-color: #464646;
+  }
+  .container.dark {
+    background-color: #262626;
+    color: white;
+  }
+  .toggle i {
+    font-size: 4rem;
+    color: #777;
+  }
+  
+  .container.dark .stopwatch .toggle i {
+    color: #ddd;
+  }
+  .container.dark .stopwatch__btns-b {
+    border: none;
+    cursor: pointer;
+    color: #faf3f3;
+    font-size: 1.75rem;
+    background-color: #0f0202;
+    padding: 10px 30px;
+    border-radius: 25px;
+    font-weight: 600;
+    box-shadow: 19px 19px 32px #0e0d0d, -19px -19px 32px #1b1b1b; }
+
+    .container.dark .copyright {
+      color: white;
+    }
+    
+    .container.dark .stopwatch__btns-stop {
+      background-color: #ffffff;
+      color: #ff4a4a;
+      box-shadow: rgba(50, 50, 93, 0.35) 0px 2px 5px -1px, rgba(0, 0, 0, 0.5) 0px 1px 3px -1px; }
+      .container.dark .stopwatch__btns-resume {
+      background-color: #ffffff;
+      color: #3ab0ff;
+      box-shadow: rgba(50, 50, 93, 0.35) 0px 2px 5px -1px, rgba(0, 0, 0, 0.5) 0px 1px 3px -1px; }
+
```

Comment:
@Farhan-Ali2002 I forgot to tell you that you have to work on the sass folder on this one, because whatever we write on the sass folder it gets compiled and sent to css, for this you need to install npm_modules and run a script.
If you don't know how to do that I can help you, it's not hard!